wrappers/ortools.rb

```diff
@@ -100,6 +99,17 @@ def solve(vrp, job, thread_proc = nil, &block)
           empty: false
         }
       }
+      sum_quantities = vrp.units.map{ |unit| [unit.id, 0] }.to_h
+      vrp.services.each{ |service|
+        service.quantities.each{ |q|
+          sum_quantities[q.unit.id] += (q.value || 0).abs
+        }
+      }
```

Comment:
This double counts the demand of the services in shipment relation. 

I think you can first do a loop on `vrp.relations.select{:shipment}.each.linked_services.last.quantities.each`
then in the `vrp.services.each` loop skip the `service.relations.any?{:shipment}`

Reply:
I think `total_quantities` or `quantity_totals` is easier to understand than `sum_quantities`.